### ISSUE TITLE
Automatic redirection general switch

### ISSUE THREAD

**Issue by khabib227** [fluks/redirect-link#33]
First thank you for implement the automatic redirection (https://github.com/fluks/redirect-link/issues/12), this is very useful. Now make sense add a little button, maybe on the side of preferences button, to toggle the automatic redirection feature and change the plugin icon to show the status, is also interesting allow a custom keyboard shortcut to toggle.

**Comment by fluks** [comment]
You mean it would enable or disable all the redirections that happen automatically?

**Comment by khabib227** [comment]
Yes, I mean it, but keeping the Always checkboxes selected, and keeping this redirections hidden on menu, as is today. The toggle should work like a pause button for all "Always" redirections. So image, if I have 8 manual redirections, and 8 automatic redirections that has the Always checked, it I use the "Pause" button, this 8 automatic redirections stop to work. If you didn't understand may explanation, let me know. And congrats and thanks for this plugin, it's getting better every day.
